src/chunkserver/copyset_service.h

```diff
@@ -30,6 +30,9 @@ namespace chunkserver {
 
 using ::google::protobuf::RpcController;
 using ::google::protobuf::Closure;
+using COPYSET_OP_STATUS::COPYSET_OP_STATUS_COPYSET_IS_HEALTHY;
```

Comment:
For legacy C enum define, its internal enumerators are already exposed to its outer namespace, you can use `COPYSET_OP_STATUS_COPYSET_IS_HEALTHY` directly without the prefix `COPYSET_OP_STATUS::`,  so no need to use those using.

Reply:
> For legacy C enum define, its internal enumerators are already exposed to its outer namespace, you can use `COPYSET_OP_STATUS_COPYSET_IS_HEALTHY` directly without the prefix `COPYSET_OP_STATUS::`, so no need to use those using.

yeap, it's a bad use of plain enums, i will improve it.